src/renderer/containers/widgets/channels/ChannelMessages.tsx

```diff
@@ -19,6 +19,8 @@ import appHandlers from '../../../store/handlers/app'
 import electronStore from '../../../../shared/electronStore'
 
 export const ChannelMessages = ({ tab, contentRect }) => {
+  const isDev = process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'production'
```

Comment:
Why isDev if process.env.NODE_ENV === 'production'?
Also it's the same line as in store/handlers/users.ts, maybe we could take this value from one place?

Reply:
my mistake